packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -215,25 +250,37 @@ class CupertinoNavigationBar extends StatefulWidget implements ObstructingPrefer
     this.trailing,
     this.border = _kDefaultNavBarBorder,
     this.backgroundColor,
+    this.background,
```

Comment:
For consistency (with classes like `Container`) I'd call this decoration rather than background.